pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -110,6 +114,8 @@ func TestProcessTicks(t *testing.T) {
 	// create alert rule under main org with one second interval
 	alertRule1 := models.AlertRuleGen(models.WithOrgID(mainOrgID), models.WithInterval(cfg.BaseInterval), models.WithTitle("rule-1"))()
 	ruleStore.PutRule(ctx, alertRule1)
+	// Assume for the sake of the test that the eventually consistent rule fetches are "fast enough."
+	fetcher.Refresh(ctx)
```

Comment:
the tests are currently run with `fetchAsync=false`, but I verified that all the tests also do pass with `fetchAsync=true`.